lib/gooddata/models/process.rb

```diff
@@ -91,6 +91,8 @@ def upload_package(path, files_to_exclude, opts = { :client => GoodData.connecti
       # @option options [String] :process_id ID of a process to be redeployed (do not set if you want to create a new process)
       # @option options [Boolean] :verbose (false) Switch on verbose mode for detailed logging
       def deploy(path, options = { :client => GoodData.client, :project => GoodData.project })
+        return deploy_component(path.merge options) if path.kind_of? Hash and path[:component_name]
```

Comment:
Style/NestedParenthesizedCalls: Add parentheses to nested method call `path.merge options`.